CampaignConfig/mdc2020_prolog.cfg

```diff
@@ -172,3 +184,7 @@ name = true
 arg_1 = --desc=%(stage_name)s
 arg_2 = --dsconf=%(desc)s
 arg_3 = --dsowner=%(submitter)s
+
+[executable_3]
+name = true
+arg_1 = %(testid)s
```

Comment:
Looks like a spurious blank line at the end: the commit-check should have caught that.